gui/screens/settings_wallet.py

```diff
@@ -101,6 +117,13 @@ def is_remove_account_button_visible(self):
     def get_account_name_value(self):
         return self._wallet_account_title.text
 
+    @allure.step('Get account balance value')
+    def get_account_balance_value(self):
+        time.sleep(3)
```

Comment:
@StateOf-Vlado @Valentina1133 there is a problem over here, when i open account details, the balance is not updated immediately, the app gets it within approximately 3 seconds. I am open to any suggestions how to improve this time.sleep. What i tried - `waitFor ( float(balance != 0.0))` but it does not help much, the test picks up the value immediately and it is always 0 then.

Reply:
I checked, replaced step in test_settings_include_in_total_balance (48-49) with

assert driver.waitFor(lambda: acc_view.get_account_balance_value() != '0,00', configs.timeouts.UI_LOAD_TIMEOUT_MSEC)

It worked for me without time.sleep, did you add timeout there? Can you try it with timeout?

Reply:
yessss will do that again, not sure why it was not working before for me 🙄

Reply:
yes, indeed, i was doing it incorrectly , i was trying to do with variable which was already stupid. Replacing the condition with function call works, thank you @Valentina1133